Packs/MailListener/Integrations/MailListenerV2/MailListenerV2_test.py

```diff
@@ -597,3 +597,33 @@ def test_replace_spaces_in_credentials(input_credentials, output_credentials):
     import json
 
     assert (json.dumps(replace_spaces_in_credentials(input_credentials)) == json.dumps(output_credentials))
+
+
+def test_fetch_incidents(mocker):
+    """
+    Given:
+        - A mock client and last run
+    When:  
+        - Fetching incidents
+    Then:
+        - Ensure that the "last_uid" received from the 'last_run' of previous cycles is converted to an integer.
+        Also, verify that the 'last_uid' to be written in the 'last_run' for the next cycle is a string.
+    """
+    from MailListenerV2 import fetch_incidents
+    mocker.patch('MailListenerV2.Email.convert_to_incident', return_value={})
+    fetch_mail_mocker = mocker.patch('MailListenerV2.fetch_mails', return_value=([mock_email()], [mock_email()], 5))
```

Comment:
Maybe add a case where it a str and stays this way

Reply:
will do